### ISSUE TITLE
Line detection via ocrd: PAGE XML is overwritten, result is empty

### ISSUE THREAD

**Issue by wrznr** [OCR-D/core#46]
Using
```shell
ocrd process -m ocrd-assets/dist/mets.xml characterize/exif segment-region/tesserocr
```
works great (i.e.  metadata and regions show up in OUTPUT PAGE XML).
However, adding line detection
```shell
ocrd process -m ocrd-assets/dist/mets.xml characterize/exif segment-region/tesserocr segment-line/tesserocr
```
results in “empty” XML:
```xml
<PcGts xmlns="http://schema.primaresearch.org/PAGE/gts/pagecontent/2017-07-15" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://schema.primaresearch.org/PAGE/gts/pagecontent/2017-07-15 http://schema.primaresearch.org/PAGE/gts/pagecontent/2017-07-15/pagecontent.xsd">
        <Page imageFileName="http://localhost:5001/00000005.tif">
        </Page>
</PcGts>
```

**Comment by wrznr** [comment]
The problem could be
```python
page = OcrdPage.from_file(self.workspace.download_file(input_file))
```
line 20 in `https://github.com/OCR-D/pyocrd/blob/master/ocrd/processor/segment_line/tesserocr.py`.

**Comment by kba** [comment]
Since this is now six months old, and the `process` subcommand has been outdated for a while, this is hard to reproduce right now. But the bug is still there and we should fix it once #199 is in and test it stays that way.
